apps/crn-server/test/integration/common/research-outputs.integration-test.ts

```diff
@@ -199,14 +199,14 @@ describe('research outputs', () => {
         const now = new Date().toISOString();
         const response = await supertest(app)
           .put(`/research-outputs/${researchOutputId}`)
-          .send({ ...researchOutput, reviewRequestedById: loggedInUser.id })
+          .send({ ...researchOutput, statusChangedById: loggedInUser.id })
           .expect(200);
 
-        expect(response.body.reviewRequestedBy.id).toEqual(loggedInUser.id);
-        expect(response.body.reviewRequestedBy.firstName).toEqual(
+        expect(response.body.statusChangedBy.id).toEqual(loggedInUser.id);
+        expect(response.body.statusChangedBy.firstName).toEqual(
           loggedInUser.firstName,
         );
-        expect(response.body.reviewRequestedBy.lastName).toEqual(
+        expect(response.body.statusChangedBy.lastName).toEqual(
```

Comment:
Should this payload also include `isInReview`? It seems like it should.

If it's inferred on the server then we _probably_ want to assert that it is inferred correctly.